app/assets/v2/js/grants/cancel_subscription.js

```diff
@@ -78,3 +78,22 @@ $(document).ready(() => {
   });
 
 });
+
+const notifyOwnerAddressMismatch = () => {
+  if (!web3 || !web3.eth)
+    return;
+  web3.eth.getAccounts((error, accounts) => {
+    if (document.contxt.github_handle == $('#subscriber-handle').val() &&
+        accounts[0] != $('#subscriber-address').val()) {
+      if ($('#cancel-subscription').attr('disabled') != 'disabled') {
+        $('#cancel-subscription').attr('disabled', 'disabled');
+        _alert({
+          message: 'Looks like you\'ve funded this grant with ' + $('#token_address').val() + '. Switch to that to take action on your subscription.'
```

Comment:
seems like this would be more appropriate as a tooltip on the cancel button instead of a giant red banner on the page when i go to it.